src/main/java/seedu/mindmymoney/helper/AddCommandInputTests.java

```diff
@@ -264,6 +264,13 @@ public static void testCreditCardLimit(String inputLimit) throws MindMyMoneyExce
             throw new MindMyMoneyException("Limit amount must be more than 0");
         }
         assert inputAmountAsDouble > 0 : "Limit amount should have a positive value";
+
+        if (inputAmountAsDouble > 50000) {
```

Comment:
If you intend to throw an error when the limit amount is at least 50,000, then should this be `>=`?

Reply:
meant to be more than 50k, fixed thanks